test/functional/Valhalla/src_lw5/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -0,0 +1,3516 @@
+/*******************************************************************************
+ * Copyright IBM Corp. and others 2018
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+ *******************************************************************************/
+package org.openj9.test.lworld;
+
+import static org.testng.Assert.*;
+import org.testng.annotations.Test;
+
+/*
+ * Instructions to run this test:
+ *
+ * 1)  Build the JDK with the '--enable-inline-types' configure flag
+ * 2)  cd [openj9-openjdk-dir]/openj9/test
+ * 3)  git clone https://github.com/adoptium/TKG.git
+ * 4)  cd TKG
+ * 5)  export TEST_JDK_HOME=[openj9-openjdk-dir]/build/linux-x86_64-server-release/images/jdk
+ * 6)  export JDK_VERSION=Valhalla
+ * 7)  export SPEC=linux_x86-64_cmprssptrs
+ * 8)  export BUILD_LIST=functional/Valhalla
+ * 9)  export AUTO_DETECT=false
+ * 10) export JDK_IMPL=openj9
+ * 11) make -f run_configure.mk && make compile && make _sanity
+ */
+
+@Test(groups = { "level.sanity" })
+public class ValueTypeTests {
+	// static Lookup lookup = MethodHandles.lookup();
```

Comment:
I see a lot of lines are commented out here. Usually we don't merge commented code. We keep the original copy of `ValueTypeTests.java` under `/src_qtypes` folder. For now, can these commented lines be removed from this file ? The deleted tests in this fie can be later replaced with newer version (with no q type dependency) gradually in the upcoming PRs.

Reply:
Sure that works.